packaging/generate_actions.py

```diff
@@ -0,0 +1,38 @@
+import jinja2
```

Comment:
`jinja2` might be too much for this particular use case: it's used only to substitute a couple of fields; also it's not a standard package. I suggest using [`str.format`][str.format] instead and reducing the dependency.

[str.format]: https://docs.python.org/3/library/stdtypes.html#str.format

Reply:
... considering these scripts do not receive strings from random users.

Reply:
OK, I removed `jinja2`